packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { concatStyleSets, IStyle, ITextFieldStyles, mergeStyles, TextField, useTheme } from '@fluentui/react';
+import React, { useState } from 'react';
+import { useLocale } from '../../localization';
+import { containerStyles, digitStyles, rowStyles, subStyles, textFieldStyles } from './Dialpad.styles';
+
+/**
+ * Strings of {@link Dialpad} that can be overridden.
+ *
+ * @public
+ */
+export interface DialpadStrings {
+  errorText: string;
+  defaultText: string;
+}
+
+/**
+ * Props for {@link Dialpad} component.
+ *
+ * @public
+ */
+export interface DialpadProps {
+  strings?: DialpadStrings;
+  containerStyles?: IStyle;
```

Comment:
Great job making all the styles overridable! Here encapsulate them all into a single property.
If you look at fluent's context menu styles for an example:
![image](https://user-images.githubusercontent.com/2684369/167489310-8611ca29-e202-4d3f-8adf-11d08739508b.png)

So for us it would be something like:
```
export interface DialpadStyles {
  root: IStyle; // instead of `container`
  digit: IStyle;
  etc...
}
```

Reply:
updated